File: app/Models/AdminData.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Laravel\Sanctum\HasApiTokens;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class AdminData extends Model
{
    use HasApiTokens, HasFactory;

    protected $fillable = [
        'daily_last_update',
        'trending_last_update',
    ];

    protected $casts = [
    'daily_last_update' => 'datetime',
    'trending_last_update' => 'datetime',
    ];

    protected $dates = ['deleted_at'];
}

File: CritiqlyAdmin/Models/AdminData.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Text.Json.Serialization;

namespace CritiqlyAdmin.Models
{
    public class AdminData
    {
        public int Id { get; set; }

        [JsonPropertyName("daily_last_update")]
        public DateTime DailyLastUpdate { get; set; }

        [JsonPropertyName("trending_last_update")]
        public DateTime TrendingLastUpdate { get; set; }
    }
}
